extensions/elation/types/allergy.ts

```diff
@@ -0,0 +1,8 @@
+import { type z } from 'zod'
+import { type allergySchema } from '../validation/allergy.zod'
```

Comment:
We usually keep this in the same file. 

Example: https://github.com/awell-health/awell-extensions/blob/ET-354-add-allergy-action-to-elation/extensions/bland/api/schema/SendCall.schema.ts